### PR TITLE
Modified into Spring MVC.

### PR DESCRIPTION
**Pull request by ArmineZak**

### CODE REVIEW

File: src/main/java/com/epam/dao/impl/QuestionDAOImpl.java [hunk 1]
```diff
@@ -15,6 +16,7 @@
 import java.util.List;
 import java.util.Map;
 
+@Service
```
Comment (SevakMart) [inline src/main/java/com/epam/dao/impl/QuestionDAOImpl.java]: @Repository

File: src/main/webapp/WEB-INF/views/process/aaronBeck.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <title>Аарон Бек</title>
+</head>
+<body>
+<h1>Пройдите опрос</h1>
+
+<form th:method="GET" th:action="@{/result/Aaron-Beck}">
```
Comment (SevakMart) [inline src/main/webapp/WEB-INF/views/process/aaronBeck.html]: in forms always use post method

**Comment by SevakMart** [comment]
don't push the target folder and .idea folder

File: src/main/java/com/epam/controller/ResultController.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    @GetMapping("/Aaron-Beck")
+    public String aaronBeckResult(@RequestParam Map<String, String> map,
+                                  Model model) {
+        int score = 0;
+        for (Map.Entry<String, String> entry : map.entrySet()) {
+            int num = Integer.parseInt(String.valueOf(entry.getValue()));
+            score += num;
+        }
+
+        if (score >= 30 && score <= 63) {
```
Comment (SevakMart) [inline src/main/java/com/epam/controller/ResultController.java]: this kind of logic usually lies in the service layer, where the main business logic is.

Comment (ArmineZak) [reply src/main/java/com/epam/controller/ResultController.java]: The changes are made and pushed. Thanks for the review.

**Pull request merged**